core/controllers/learner_group.py

```diff
@@ -774,7 +858,22 @@ def get(self, learner_group_id: str) -> None:
         })
 
 
-class LearnerGroupLearnerInvitationHandler(base.BaseHandler):
+class LearnerGroupLearnerInvitationHandlerNormalizedPayloadDict(TypedDict):
+    """Dict representation of LearnerGroupLearnerInvitationHandler's
+    normalized_payload dictionary.
+    """
+
+    learner_username: str
+    is_invitation_accepted: Union[str, bool]
+    progress_sharing_permission: Union[str, bool]
```

Comment:
This is weird, also the schema is probably incorrect, we should pass false as a bool and not as a string.

Reply:
Feel free to ask the author of these handlers to fix this.

Reply:
Any update on this? At least TODO should be added.

Reply:
> Any update on this?

@paprajapati9 will fix this, but currently, he is busy with some other work. So todo is added #16566